torch/linalg/__init__.py

```diff
@@ -1492,6 +1492,8 @@
              The gradient will also be numerically unstable when :attr:`A` has small singular
              values, as it also depends on the computaiton of :math:`\frac{1}{\sigma_i}`.
 
+.. warning:: Backpropagation is only supported for input matrices with at most one zero eigenvalue.
```

Comment:
I feel like the warning above actually mentions most of this already. Sorry for missing that.
We should fix the formatting of `Vh` though to be `V` or `V^H` with math formatting.

Reply:
it looks like a similar warning exists in both variants of the function. So revert changes to both files and edit this file to use V^H instead of Vh? Should I do that across the board for this function or just the nearest warning?

Reply:
I see some place already using Vᴴ. How does this look?